Sources/TuistSupport/Errors/FatalError.swift

```diff
@@ -30,7 +30,7 @@ public struct UnhandledError: FatalError {
     public let type: ErrorType = .bug
 
     /// Error description.
-    public var description: String { error.localizedDescription }
+    public var description: String { "UnhandledError - \(error.localizedDescription)\n\(error)" }
```

Comment:
This might end up printed to the user. What about making it more readable?

```
We received an error that we couldn't handle:
  - Localized description: xxx
  - Error: yyy
```

Reply:
Ah, good idea - the reason why I did this is because sometimes the `localizedDescription` doesn't contain enough information to figure out what's gone wrong. Whilst I was adding verbose mode I thought it would also be useful to solve this too.

We should now no longer see obscure exit code error messages, but more detailed ones. 🎉